.githooks/pre-commit/pre-commit.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "[Pre-commit] Writing prettier 🚀🚀"
+
+npm run prettier
+
+CHANGED_FILES=$(git diff)
```

Comment:
what if I change ".gitignore" file? this script still runs. Prettier is suppose to be run only for "*.ts" "*.tsx" "*.js" "*.jsx" "*.md" ".less"

How about this?
```suggestion
CHANGED_FILES =$(git diff --cached --name-only --diff-filter=ACMR "*.ts" "*.tsx" "*.js" "*.jsx" "*.md" | sed 's| |\\ |g')
```

Reply:
This one should start first before running any scripts. Consider this algorithm

```bash
if no changed files:
   exit
then run prettier here
```